pkg/daemon/file_writers.go

```diff
@@ -45,12 +46,80 @@ func noOrigFileStampName(fpath string) string {
 	return filepath.Join(noOrigParentDir(), fpath+".mcdnoorig")
 }
 
+func isFileOwnedByRPMPkg(fpath string) (bool, bool, error) {
+	// The first bool returned indicated whether the underlying OS is Mac
```

Comment:
nit: for clarity purposes, maybe something like:

`The first bool returns false if rpm exists, and true otherwise (indicating other Linux distros or Mac). We would like to skip orig file creation and preservation`